jdaviz/app.py

```diff
@@ -2317,11 +2317,15 @@ def _on_new_viewer(self, msg, vid=None, name=None, add_layers_to_viewer=False):
             viewer=viewer, vid=vid, name=name, reference=name
         )
 
-        ref_data = self._jdaviz_helper.default_viewer._obj.state.reference_data
-        new_viewer_item['reference_data_label'] = getattr(ref_data, 'label', None)
-
-        if hasattr(viewer, 'reference'):
-            viewer.state.reference_data = ref_data
+        if self.config == 'imviz':
```

Comment:
I did try to generalize this logic instead of hardcoding the config, but not sure its worth it until/unless other configs need this logic?  See the commit history to see a failed attempt.